src/test/features/response/routes/statement-of-means/supported-by-you.ts

```diff
@@ -0,0 +1,150 @@
+import * as idamServiceMock from '../../../../http-mocks/idam'
+import * as draftStoreServiceMock from '../../../../http-mocks/draft-store'
+import * as claimStoreServiceMock from '../../../../http-mocks/claim-store'
+import { attachDefaultHooks } from '../../../../routes/hooks'
+import { checkAuthorizationGuards } from '../checks/authorization-check'
+import { checkAlreadySubmittedGuard } from '../checks/already-submitted-check'
+import { checkCountyCourtJudgmentRequestedGuard } from '../checks/ccj-requested-check'
+import { app } from '../../../../../main/app'
+import { checkNotDefendantInCaseGuard } from '../checks/not-defendant-in-case-check'
+
+const cookieName: string = config.get<string>('session.cookieName')
+const pagePath: string = StatementOfMeansPaths.supportedByYouPage.evaluateUri(
+  { externalId: claimStoreServiceMock.sampleClaimObj.externalId }
+)
+
+describe('Defendant response: Statement of means: do you support anyone', () => {
+
+  attachDefaultHooks(app)
+
+  describe('on GET', () => {
+
+    const method = 'get'
+    checkAuthorizationGuards(app, method, pagePath)
+    checkNotDefendantInCaseGuard(app, method, pagePath)
+
+    context('when user authorised', () => {
+
+      beforeEach(() => {
+        idamServiceMock.resolveRetrieveUserFor(claimStoreServiceMock.sampleClaimObj.defendantId, 'cmc-private-beta', 'defendant')
+      })
+
+      checkAlreadySubmittedGuard(app, method, pagePath)
+      checkCountyCourtJudgmentRequestedGuard(app, method, pagePath)
+
+      context('when response and CCJ not submitted', () => {
+
+        it('should return 500 and render error page when cannot retrieve claim', async () => {
+          claimStoreServiceMock.rejectRetrieveClaimByExternalId('HTTP error')
+
+          await request(app)
+            .get(pagePath)
+            .set('Cookie', `${cookieName}=ABC`)
+            .expect(res => expect(res).to.be.serverError.withText('Error'))
+        })
+
+        it('should return 500 and render error page when cannot retrieve draft', async () => {
+          claimStoreServiceMock.resolveRetrieveClaimByExternalId()
+          draftStoreServiceMock.rejectFind('Error')
+
+          await request(app)
+            .get(pagePath)
+            .set('Cookie', `${cookieName}=ABC`)
+            .expect(res => expect(res).to.be.serverError.withText('Error'))
+        })
+
+        it('should render page when everything is fine', async () => {
+          claimStoreServiceMock.resolveRetrieveClaimByExternalId()
+          draftStoreServiceMock.resolveFind('response')
+
+          await request(app)
+            .get(pagePath)
+            .set('Cookie', `${cookieName}=ABC`)
+            .expect(res => expect(res).to.be.successful.withText('Do you support anyone else financially?'))
+        })
+      })
+    })
+  })
+
+  describe('on POST', () => {
+
+    const method = 'post'
+    checkAuthorizationGuards(app, method, pagePath)
+    checkNotDefendantInCaseGuard(app, method, pagePath)
```

Comment:
Add `checkNotDefendantInCaseGuard` guard as well. Here and for POST tests.

Reply:
done